src/pkg/services/m365/sites_test.go

```diff
@@ -216,3 +222,146 @@ func (suite *siteUnitSuite) TestGetAllSites() {
 		})
 	}
 }
+
+func userIdentitySet(userID string, userEmail string) models.IdentitySetable {
```

Comment:
the `-Set` suffix is kinda confusing.  I think the names could do without.
```suggestion
func userIdentity(userID string, userEmail string) models.IdentitySetable {
```